### PR TITLE
remove axios retries and cleanup logs

### PR DESCRIPTION
**Pull request by DeveloperInProgress**
# Description
`retryOnFailAxios` combined with retries from ConnectionPool class will multiply the number of retries. This can be fixed by removing `retryOnFailAxios`

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I have tested locally
- [ ] I have performed a self review of my changes
- [ ] Updated any relevant documentation
- [ ] Linked to any relevant issues
- [ ] I have added tests relevant to my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] My code is up to date with the base branch

### CODE REVIEW

File: packages/node/src/utils/cosmos.ts [hunk 1]
```diff
@@ -189,14 +189,10 @@ async function getBlockByHeight(
 ): Promise<[BlockResponse, BlockResultsResponse]> {
   return Promise.all([
     api.blockInfo(height).catch((e) => {
-      const error = CosmosClient.handleError(e);
-      logger.error(error, `failed to fetch block info ${height}`);
-      throw error;
+      throw CosmosClient.handleError(e);
```
Comment (stwiname) [inline packages/node/src/utils/cosmos.ts:192]: What's the reason for these changes? Logging the reason it failed is useful

Comment (DeveloperInProgress) [reply packages/node/src/utils/cosmos.ts:192]: too much logs. For example, I noticed we log rate limit on every single block in a batch (about 100 times by default)

**Review by stwiname — APPROVED** [review]
